src/Commands/core/SshCommands.php

```diff
@@ -49,7 +50,10 @@ public function ssh(array $code, $options = ['cd' => self::REQ, 'tty' => false])
         }
 
         $process = $this->processManager()->siteProcess($alias, $code);
-        $process->setTty($options['tty']);
+        if (Tty::isTtySupported()) {
```

Comment:
We still need to allow the user to select whether a TTY is desired or not. The other end of the connection might not support TTYs. I think that the `tty` CLI option should be preserved, and the `$process->setTty($options['tty']);` should merely be wrapped in `if (Tty::isTtySupported()) {`

Reply:
Ah right. I pushed a commit which does this. 

The $options for ssh method did not need to declare 'tty' key since thats done by `@optionset_proc_build`